tensilelite/Tensile/Components/GlobalWriteBatch.py

```diff
@@ -2008,7 +2008,8 @@ def _addSumAlphaWithCBeta(self, kernel, ss, gwvw, elementIdx, vc0, tmpVgpr, cvtV
               sb = 0 if vi ==0 else 1
               module.add(VCvtPkFP8toF32(dst=vgpr(tmpVgpr, 2), src=vgpr(dataV), vop3=VOP3PModifiers(op_sel=[sb])))
             else:
-              sb = SelectBit.WORD_0 if vi == 0 else SelectBit.WORD_1
+              # Enable WORD_0 of 2-nd VGPR with vi=4 for vw=8
+              sb = SelectBit.WORD_0 if vi%4 == 0 else SelectBit.WORD_1
```

Comment:
Don't we need to apply the same logic for NoSDWA case at line 2009?

Reply:
Sure. Done, Please approve it.

Reply:
Please help to remove #cm review

Reply:
Done